src/User32.Desktop/User32.cs

```diff
@@ -144,6 +144,17 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern unsafe IntPtr SendMessage(IntPtr hWnd, int wMsg, void* wParam, void* lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern unsafe IntPtr SendMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
```

Comment:
We should reflect on what to do here to have a single overload (Potentially taking an enum with some implicit cast from int to allow custom messages, I don't know just throwing ideas around there)